### PR TITLE
bt-468: Add candidate cabinet

### LINKED ISSUE #468: Create candidate cabinet for profile updating
Cabinet will be in user dropdown menu

Image

Image

**User story**

As a Talent, I want to have access to my personal cabinet, so that I can edit my personal info

**Acceptance criteria:**
1. The user can go to the page of the edit personal information after choosing this option in the dropdown component under the user avatar
2. The information displayed on this page must meet the information that the user filled in during the registration
3. The content area of the page must contain:
- the title “Your profile”
- the tabs “Profile”, “BSA badges”, “Skills and projects”, “CV and contacts”, “Hiring”
- the active buttons “Save”, “Edit” 

4. The content area of the tab “Profile” must contain the fields with the next titles:
- “Profile name”
- “Salary  expectations”
- “Job title”
- “Experience”
- “Current location”
- “Employment type”
- “Briefly tell employers about your experience” 

5. The data that can be entered into the fields “Profile name”,  “Salary  expectations”
“Job title”, “Briefly tell employers about your experience”  should meet the criteria that are written in the [validation document](https://docs.google.com/spreadsheets/d/1OM1GunJPbMa6u0Dn4rm9Cgh32PBfMNOxv588UZYBM_4/edit?usp=sharing).

### LINKED ISSUE #469: Create employer cabinet for profile updating
**User story:**
As an employer, I want to have a profile screen, so that I can edit my profile information and submit it for verification to the admin

**Acceptance criteria:**

- There should be a user avatar in the right upper corner 
- When a user clicks on avatar the menu with logout appears
- There should be a sidebar on the left side with the Chats and Candidates buttons
- There should be tabs: Profile and Hirings
- Profile tab should have: Full name, Your position, Linkedin profile, Company name field, Company website,  Briefly tell about your company and it’s value fields [Fields validation](https://docs.google.com/spreadsheets/d/1OM1GunJPbMa6u0Dn4rm9Cgh32PBfMNOxv588UZYBM_4/edit#gid=0)
- Hirings tab should display a list of candidates that which employer hired
- There should be fields for profile photo and company logo

Image

Image

### PR DESCRIPTION
**Pull request by savinaDaria**
- Also contains employer cabinet
- Added actions in store for proper saving of employer data 

image
image
image
image

### CODE REVIEW

File: frontend/src/bundles/common/enums/app-route.enum.ts [hunk 1]
```diff
@@ -9,6 +9,7 @@ const AppRoute = {
     EMPLOYER_ONBOARDING: '/employer/onboarding',
     TALENT_STEP: '/talent/onboarding/step/:step',
     NOT_FOUND: 'not-found',
+    MY_PROFILE: '/my/:step',
```
Comment (nika-sirak) [inline frontend/src/bundles/common/enums/app-route.enum.ts]: Maybe it's better to write 'my-cabinet' or 'my-card'

Comment (savinaDaria) [reply frontend/src/bundles/common/enums/app-route.enum.ts]: @nika-sirak maybe
I've chosen my/.. as in djinni

Comment (nikita-remeslov) [reply frontend/src/bundles/common/enums/app-route.enum.ts]: `my-profile` will be good

**Comment by nikita-remeslov** [comment]
Conflicts

**Review by nikita-remeslov — APPROVED** [review]

**Pull request merged**